src/vs/workbench/contrib/markdown/common/markdownDocumentRenderer.ts

```diff
@@ -27,6 +27,16 @@ img {
 	max-height: 100%;
 }
 
+svg {
```

Comment:
@JacksonKearl not sure if you know what the implications are for setting this at the root level for the markdown doc renderer (not sure where else we use this). I tested this on our markdown preview and didn't see any side effects.

Reply:
Yes, it should instead be injected here to be as narrowly scoped as possible: https://github.com/microsoft/vscode/blob/023b38e89d8a1cd188b5d9ac014332af3cc27d04/src/vs/workbench/contrib/welcome/gettingStarted/browser/gettingStarted.ts#L665

Reply:
Moved per Jackson's advice

Reply:
Updated syntax highlighting, diff insertion/removal icon theming, and list highlight colors per feedback above. Thanks for the feedback!

Leaving the issue re: shadows on light themes for a follow-up PR since I think it's a more involved change.